manifests/init.pp

```diff
@@ -325,9 +329,11 @@
       register_in_foreman   => $register_in_foreman,
       foreman_base_url      => $foreman_url,
       registered_proxy_url  => "https://${capsule_fqdn}:${capsule::foreman_proxy_port}",
+      trusted_hosts         => undef,
```

Comment:
Then it would be probably better to keep that in separate commit with the proper reference number